src/encoded/static/components/body_map.js

```diff
@@ -602,24 +613,25 @@ class BodyMap extends React.Component {
         return (
             <div className="body-facet-container">
                 <div className="body-list body-list-top">
-                    <div className="body-list-inner">
+                    <ul className="body-list-inner">
                         {Object.keys(SystemsList).map(b =>
-                            <button
-                                className={`body-list-element ${((typeof this.state.selectedOrgan === 'string' && this.state.selectedOrgan === b) || (typeof this.state.selectedOrgan !== 'string' && this.state.selectedOrgan.includes(b))) ? 'active' : ''}`}
-                                id={b}
-                                onClick={e => this.chooseOrgan(e)}
-                                onMouseEnter={e => highlightOrgan(e)}
-                                onMouseLeave={unHighlightOrgan}
-                                key={b}
-                            >
-                                {b}
-                            </button>
+                            <li key={b}>
+                                <button
+                                    className={`body-list-element ${((typeof this.state.selectedOrgan === 'string' && this.state.selectedOrgan === b) || (typeof this.state.selectedOrgan !== 'string' && this.state.selectedOrgan.includes(b))) ? 'active' : ''}`}
+                                    id={b}
+                                    onClick={e => this.chooseOrgan(e)}
+                                    onMouseEnter={e => highlightOrgan(e)}
```

Comment:
Why don't highlightOrgan and unHighlighOrgan have "this." in front? They both are within a React Class component.

Reply:
I think they're actually not in React Class component and don't use `this`, though I might be missing something.